CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [FEATURE] Enable DatadogCore, DatadogLogs and DatadogTrace to compile on watchOS platform. See [#1918][] (Thanks [@jfiser-paylocity][]) [#1946][] 
 - [IMPROVEMENT] Ability to clear feature data storage using `clearAllData` API. See [#1940][]
+- [IMPROVEMENT] Send memory warning as RUM error.
```

Comment:
change-request/ We're missing PR ID here, making it harder to combine release notes when shipping next release.

Reply:
Fixed.